public/server/db/pc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "definition":"Personal Computer"
```

Comment:
Abbreve is an unofficial dictionary for slang. Unfortunately, abbreviations like these don't fit in. Hope you understand.

Reply:
Sure I make a PR again for this keeping in mind the changes suggested . Thanks